src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml

```diff
@@ -51,7 +51,7 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="2"/>
-    <attribute side="server" code="0x0000" define="IDENTIFY_TIME" type="INT16U" min="0x00" max="0xFE" writable="true"  default="0x0" optional="false">identify time</attribute>
+    <attribute side="server" code="0x0000" define="IDENTIFY_TIME" type="INT16U" min="0x00" max="0xFFFE" writable="true"  default="0x0" optional="false">identify time</attribute>
```

Comment:
The right max per the spec is 0XFFFF, so we should actually just remove the min/max bits here altogether:
```suggestion
    <attribute side="server" code="0x0000" define="IDENTIFY_TIME" type="INT16U" writable="true"  default="0x0" optional="false">identify time</attribute>
```

Reply:
Added this change and regenerated zap